libraries/AP_BattMonitor/AP_BattMonitor_SMBus_Maxell.cpp

```diff
@@ -0,0 +1,94 @@
+#include <AP_HAL/AP_HAL.h>
+#include <AP_Common/AP_Common.h>
+#include <AP_Math/AP_Math.h>
+#include "AP_BattMonitor.h"
+#include "AP_BattMonitor_SMBus_Maxell.h"
+#include <utility>
+
+extern const AP_HAL::HAL& hal;
+
+#include <AP_HAL/AP_HAL.h>
+
+#define BATTMONITOR_SMBUS_TEMP      0x08    // temperature register
```

Comment:
@muramura, thanks. I've added two commits that make the register definitions specific to the class.  I could have moved the Voltage definition up to the parent SMBus class but it didn't seem helpful to have the register definitions in two places.